### PR TITLE
Clustered installation docs

### PR DESCRIPTION
**Pull request by sanderson**
Adds installation and configuration docs to InfluxDB Clustered.

@jstirnaman most of these changes were reviewed before being merged into the `clustered-install` branch, so it's likely good to go, but it may benefit from a quick once-over.

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/clustered/install/configure-cluster.md [hunk 1]
```diff
@@ -0,0 +1,778 @@
...
+  _(See [Find user IDs with Microsoft Entra ID](/influxdb/clustered/install/auth/?t=Microsoft+Entra+ID#find-user-ids-with-microsoft-entra-id))_
+
+---
+
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+##### Adding users
+
+Finally, you will need to add all the users you wish to have access to use influxctl.
```
Comment (jstirnaman) [inline content/influxdb/clustered/install/configure-cluster.md:684]:
```suggestion
Finally, add all the users you wish to have access to use `influxctl`.
```

File: content/influxdb/clustered/install/configure-cluster.md [hunk 2]
```diff
@@ -0,0 +1,778 @@
...
+Update the `spec.package.spec.admin.users` field with a list of these users.
+See [Adding or removing users](/repalce/this) for more details.
```
Comment (jstirnaman) [inline content/influxdb/clustered/install/configure-cluster.md:686]:
```suggestion
See [Adding or removing users](/influxdb/clustered/admin/users/) for more details.
```

I assume?

**Review by jstirnaman — CHANGES_REQUESTED** [review]
TIL!

File: content/influxdb/clustered/install/auth.md [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+
+### Find user IDs with Auth0
+
+For Auth0, you can find the user ID through the web UI.
+Log in to your tenant and navigate to User Management.
+From there, search for the user.
+Once found, the user ID can be found at the top of the page under the user's name. -->
+
+<!--------------------------------- END Auth0 --------------------------------->
+{{% /tab-content %}}
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/clustered/install/auth.md:194]: Ah, this was meant as placeholder and should've been commented out. Thanks for catching. I'll ping the clustered team to see if they can provide instructions on using Auth0.

File: content/influxdb/clustered/install/configure-cluster.md [hunk 4]
```diff
@@ -0,0 +1,778 @@
...
+  influxdb_clustered:
+    name: Configure your cluster
+    parent: Install InfluxDB Clustered
+weight: 103
+---
+
+InfluxDB Clustered deployments are managed with Kubernetes and configured using
+a YAML configuration file. InfluxData provides the following items:
+
+- **`influxdb-docker-config.json`**: Authenticated Docker configuration file.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/clustered/install/configure-cluster.md]: I believe it's controlled by an auth token. As I understand it, we provide the customer with a Docker config with pre-populated settings, including a token or some type of auth to pull the private images required to provision a cluster.

File: content/influxdb/clustered/install/configure-cluster.md [hunk 5]
```diff
@@ -0,0 +1,778 @@
...
+
+InfluxDB ingesters require local storage to store the Write Ahead Log (WAL) for
+incoming data.
+To connect your InfluxDB cluster to local storage, provide values for the
+following fields in your `myinfluxdb.yml` configuration file:
+
+- `spec.package.spec.ingesterStorage`
+  - `.storageClassName`: [Kubernetes storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/).
+    This will differ based on the Kubernetes environment and desired storage characteristics.
+  - `storage`: Storage size. We recommend a minimum of 2 gibibytes (`2Gi`).
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/clustered/install/configure-cluster.md:529]: Yes, this format is specific to the config: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory

File: content/influxdb/clustered/install/prerequisites.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+  - 1 m6i.2xlarge instance for compactors
+  - 1 t3.large for the Kubernetes control plane
+
+Your sizing may need to be different based on your environment and workload,
+but this is a reasonable starting size for your initial testing.
+
+## Set up local storage
+
+The InfluxDB ingester pods need local storage to store the Write-Ahead Log (WAL).
+The recommended minimum size of the local storage is 2 gibibytes (`2Gi`).
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/clustered/install/prerequisites.md:71]: Yep, specific to Kubernetes configurations and defining resources available to pods.

**Pull request merged**